core/dbt/cli/main.py

```diff
@@ -576,6 +577,48 @@ def run(ctx, **kwargs):
     return results, success
 
 
+# dbt run
+@cli.command("retry")
+@click.pass_context
+@p.defer
```

Comment:
I think most of the params listed here should not be supported as retry will reconstruct them from previous run result?